static/openapi/vwrs-spec.json

```diff
@@ -1164,6 +1228,194 @@
         }
       }
     },
+    "/api/vwr/v1/requests/status/{request_id}": {
+      "get": {
+        "summary": "Position in waiting room",
+        "description": "Get the user's current position in waiting room.",
+        "tags": [
+          "Waiting room"
+        ],
+        "security": [
+          {
+            "APIKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "request_id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique request ID of a user."
+          },
+          {
+            "in": "query",
+            "name": "domain",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The domain_key of the domain."
+          },
+          {
+            "in": "query",
+            "name": "sid",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The position ID in the waiting room."
+          },
+          {
+            "in": "query",
+            "name": "created_at",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The creation timestamp when user first time entered the waiting room."
```

Comment:
```suggestion
            "description": "The creation timestamp when the user first time entered the waiting room."
```

Reply:
I'll leave the second one for now. First sounds a bit like "the timestamp when unix was created"